packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -63,7 +64,7 @@ export const Table = memo((props: Props) => {
   const [footerItems, setFooterItems] = useState<FooterItem[] | undefined>(footerValues);
 
   const footerHeight = useMemo(() => {
-    const EXTENDED_ROW_HEIGHT = headerHeight;
```

Comment:
This calculation of the footer height was being based on the headerHeight, which is fine, if the headers are being rendered/toggled on. However, if they aren't, then the value of the return of this function will always be 0, and therefore the footer values will render over table row values.